apps/frontend/src/components/global/groups/GroupManagement.vue

```diff
@@ -35,6 +35,51 @@
           />
         </v-container>
       </template>
+      <template #[`item.users`]="{item}">
+        <v-chip
+          :active="displayChip(0, 1, item.users)"
+          pill
+          color="primary"
+          class="ma-1"
+          v-on="on"
+          @click="displayMembersDialog(item.users[0])"
+        >
+          {{ getMemberNames(item.users)[0].identifier }}
+        </v-chip>
+        <v-chip
+          v-if="item.users.length >= 2"
+          :active="displayChip(992, 2, item.users)"
+          pill
+          color="primary"
+          class="ma-1"
+          v-on="on"
+          @click="displayMembersDialog(item.users[1])"
+        >
+          {{ getMemberNames(item.users)[1].identifier }}
+        </v-chip>
+        <v-chip
+          v-if="item.users.length >= 3"
+          :active="displayChip(1200, 3, item.users)"
+          pill
+          color="primary"
+          class="ma-1"
+          v-on="on"
+          @click="displayMembersDialog(item.users[2])"
+        >
+          {{ getMemberNames(item.users)[2].identifier }}
+        </v-chip>
+        <v-chip
+          :active="displayTotalMembersChip(item.users.length)"
+          pill
+          color="primary"
+          outlined
+          class="ma-1"
+          v-on="on"
+          @click="displayMembersDialog(item.users)"
```

Comment:
I'm fine with just showing the `displayMembersDialog` all the time instead of filtering the displayMembersDialog down to the specific user they clicked on.